content/derisking-government-tech/resources/quality-indicators.md

```diff
@@ -1,9 +1,71 @@
 ---
 title: Sample Quality Assurance Surveillance Plan (QASP)
 permalink: /derisking-government-tech/resources/quality-indicators/
-layout: layouts/page
+layout: layouts/wide
 tags: derisking-government-tech
+description: Sample Quality Assurance Surveillance Plan (QASP) for a performance-based services contract to build custom software for a government agency.
 sidenav: false
 ---
 
-TK
+{% capture documentedCodeStandard %}
+All dependencies are listed and the licenses are documented Major functionality in the software/source code is documented in plain language
+
+Individual methods are documented in-line using comments that permit the use of documentation generation tools such as [JSDoc]({{ "https://jsdoc.app/" | url }})
+
+A system diagram is provided
+{% endcapture %}
+
+<table class="width-full">
+  <thead>
+    <tr>
+      <th class="width-25-percent">Quality indicator for</th>
+      <th class="width-25-percent">Performance standard</th>
+      <th class="width-25-percent">Acceptable quality level</th>
+      <th class="width-25-percent">Method of assessment</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Tested code</td>
+      <td>Code delivered under the order must have substantial test code coverage and a clean code base</td>
+      <td>Minimum of 90% test coverage of all code</td>
+      <td>Automated testing</td>
+    </tr>
+    <tr>
+      <td>Properly styled code</td>
+      <td>Meets acceptable quality level</td>
+      <td>0 linting errors and 0 warnings</td>
+      <td>Styling standards and linters</td>
+    </tr>
+    <tr>
+      <td>Accessibility</td>
+      <td><a href={{ "https://www.w3.org/TR/WCAG21/" | url }}>Web Content Accessibility Guidelines 2.2 – ‘AA’ standards</a></td>
```

Comment:
The link is still pointing to WCAG 2.1, but should point to https://www.w3.org/TR/WCAG22/

Reply:
Good catch, I'll update that.